pydarn/io/superdarn_formats.py

```diff
@@ -3,6 +3,11 @@
 """
 This file contains several class pertaining to dmap format types
 that are used by SuperDARN.
+
+For description of these fields and their units see the following reference
+materials:
+    https://superdarn.github.io/rst/superdarn/src.doc/rfc/index.html
```

Comment:
I think we already know RST documentation is severally outdated ... that is an RST revamp documentation problem. I don't think pyDARN should have to deal with that.

Reply:
Okay, I just think this type of documentation should be somewhere since this IO stuff is extremely important. I don't think we should be too reliant on RST staying up to date. This can be done later.